docs/catalog/cloud/scale-infrastructure-with-user-load.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: franziska-nttdata-hesselfeld
+published_date: TBD
+category: cloud
+tags: 
+ - compute
+ - role:cloud-engineer
+ - size:medium
+---
+
+# Scale infrastructure with user load #
+
+## Description
+
+This pattern involves reviewing workloads for predictable patterns and identifying periods of low or no activity where resources can be scaled back.
+
+## Solution
+
+Scaling back resources during periods of low utilisation eliminates excess capacity and improves efficiency because a unit running at a high utilisation produces energy more efficiently. In addition, reducing the total number of resources used also reduces the amount of embodied carbon.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, scale infrastructure with user laod, will impact two parts:
+
+- `M`: Can reduce the embodied emissions if the application after the optimization uses less hardware overall
+- `E`: Optimizing the application will lead to less electricity consumed over time
```

Comment:
nit: We try to keep the E, I, M, and R references in the same order as the equation. Can you flip the E and M entries?

Reply:
Done